app/controllers/api/v1/wizards_controller.rb

```diff
@@ -24,6 +24,6 @@ def show
   protected
 
   def wizard_params
-    params.require(:wizard).permit(:name, :description, :img_url, :creator_id)
+    params.require(:wizard).permit(:id, :name, :description, :img_url, :creator_id)
```

Comment:
we do not want to allow the user to provide the id / primary key here. this should be left out of strong params